### DISCUSSION TITLE
I deleted both system_a and system_b partitions - now it won't boot ? (ugh)

### DISCUSSION THREAD

**Discussion by struesda** [Archfx/duo-de#100] [General]
I have a Surface Duo., and once I figured out how to unlock my device, the fastboot commands were working fine ... until ...

``
c:\Executables\Android_adb\platform-tools-latest-windows\platform-tools>fastboot flash system aosp-arm64-ab-gapps-15.0-20241108.img
Sending sparse 'system' 1/15 (262112 KB)           OKAY [  6.825s]
Writing 'system'                                   **FAILED (remote: 'No such file or directory')**
fastboot: error: Command failed
``

so - I wondered if I was supposed to delete both partitions - so I did.

now when I run the flash command - I get:

``
c:\Executables\Android_adb\platform-tools-latest-windows\platform-tools>fastboot flash system aosp-arm64-ab-gapps-15.0-20241108.img
Sending sparse 'system' 1/15 (262112 KB)           OKAY [  6.825s]
Writing 'system'                                   **FAILED (remote: 'No such file or directory')**
fastboot: error: Command failed
``
so ... I assume it needed one of those partitions?   Can I just create it again?  What size it is supposed to be?

**Comment by struesda** [comment]
sorry - saw the other thread about the space error - I was able to resolve it using those steps

**Comment by struesda** [comment]
and I was wrong - still not working.  Here's what I did:

1.  deleted the other system_ext_ / product_ partitions as advised in the other thread
2. got the same 'no such file or directory' error
3. I figured I would have to create that system partition after all (since I had deleted both system_a and system_b
4. I tried that create-logical-partition system 0
5. The I tried flashing again ... that suceeded.
6. But now - when I reboot - it just goes right back to the bootloader - and if I start from the bootloader - it just goes right back to the bootloader.

So now what - how do I get it to properly boot back into the newly flashed system partition :(

**Reply by Archfx** [reply]
Can you include the flash log (terminal text), while performing the steps in  [here](https://github.com/Archfx/duo-de/discussions/36)?

**Reply by billymild** [reply]
This fix above worked for me. Thank you!!!!

fastboot create-logical-partition system 0 

I then did a reflash and BOOM it started installing. THANK YOU!!!!!
